### PR TITLE
Add missing keybindings for view commands

### LINKED ISSUE #2837: Missing 'View' keybindings
There are missing keybindings for `View` commands. (Ex: `Outline`, `Plugins`, `Search`)

screen shot 2018-09-10 at 7 38 53 pm

### PR DESCRIPTION
**Pull request by vince-fugnitto**
Fixes #2837

Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
We should probably agree on which keybinding is appropriate for each command.

File: packages/outline-view/src/browser/outline-view-contribution.ts [hunk 1]
```diff
@@ -32,7 +32,8 @@ export class OutlineViewContribution extends AbstractViewContribution<OutlineVie
                 area: 'right',
                 rank: 500
             },
-            toggleCommandId: 'outlineView:toggle'
+            toggleCommandId: 'outlineView:toggle',
+            toggleKeybinding: 'ctrlcmd+shift+o'
```
Comment (akosyakov) [inline packages/outline-view/src/browser/outline-view-contribution.ts]: that's a keybinding for `Go to Symbol`

please revert it, we fixed it not so long time ago: https://github.com/theia-ide/theia/issues/2154

also please make sure for all introduced keybindings that they don't break some existing keybindings

Comment (vince-fugnitto) [reply packages/outline-view/src/browser/outline-view-contribution.ts]: thanks and sorry about that, I'll make the appropriate changes

**Comment by vince-fugnitto** [comment]
@kittaakos removed the keybinding for `Search` based on your comment

**Comment by kittaakos** [comment]
I am doing the review now...

**Comment by vince-fugnitto** [comment]
@kittaakos the only reason was either they were already being used and would collide, or they would not work in the browser so I took any available keybinding left.

**Pull request merged**